AnimateQuotes.js

```diff
@@ -82,7 +91,7 @@ export default function AnimateQuotes(props) {
       height: '100%',
       width: '100%',
     }}>
-      {props.children}
+      <Quotes quoteIndex={quoteIndex} setQuoteIndex={setQuoteIndex} />
```

Comment:
Previously, `Quotes` component was returned in `App` component nested within `AnimateQuotes` component while using `props.children` here, but choosing to instead return `Quotes` component here within `AnimateQuotes` for simplicity of code and passing state.